includes/REST_API/Stories_Controller.php

```diff
@@ -459,6 +437,8 @@ protected function prepare_links( $post ): array {
 		$links = parent::prepare_links( $post );
 		add_post_type_support( Story_Post_Type::POST_TYPE_SLUG, 'revisions' );
 
+		// Post Locking.
+
```

Comment:
```suggestion
```

Reply:
Hmm I think I'll actually just extract the whole post locking part here into its own method and remove this comment entirely.